db/seeds/base/process_statements.rb

```diff
@@ -0,0 +1,29 @@
+helpers = Class.new { include ActiveSupport::Testing::TimeHelpers }.new
+
+Cohort.all.find_each do |cohort|
+  LeadProvider.all.find_each do |lead_provider|
+    statement_scope  = Statement.where(cohort:, lead_provider:).order(deadline_date: :desc)
+    latest_statement = statement_scope.with_output_fee.where("deadline_date < ?", Time.zone.today).first
+
+    next unless latest_statement
+
+    # Mark past statements (up to latest output fee statement) as paid
+    statement_scope.where("deadline_date < ?", latest_statement.deadline_date).find_each do |statement|
+      Statements::MarkAsPayable.new(statement:).mark
```

Comment:
Can we generate some payable statements ? Or is that difficult because of timings?